src/Internal.AspNetCore.Sdk/build/AssemblyInfo.Serviceable.cs

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+[assembly: System.Reflection.AssemblyMetadata("Serviceable", "True")]
```

Comment:
Is it needed? Oh wait, is this for all our regular projects to consume (i.e. we can delete this from all the other repos)? Or is this actually compiled into a DLL right here?